tests/test_cli.py

```diff
@@ -118,7 +118,13 @@ def test_cli_directory_argument(action, mocker, capsys):
 
 @pytest.mark.parametrize("action", ENVIRONMENT_ACTIONS)
 def test_environment_actions_verbose_true(action, mocker, project_directory_factory):
-    mocked_action = mocker.patch(f"conda_project.project.Environment.{action}")
+    if action == "init":
```

Comment:
This is a little confusing to me right now. I'll need to look at what's happening in detail. Should line 129 do `if action in ["create", "init"]`?